ca_gazebo/worlds/ca.world

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" ?>
+<sdf version="1.4">
+  <world name="default">
+    <!-- <gui>
+    <camera name="gzclient_camera">
+      <pose>-7.0 0 7.0 1.5 1.5 0</pose>
```

Comment:
Seems like at startup the camera pose is looking directly into some walls. A more top-down view would be a nicer default.